quantum/nodes/not-gate/not-gate.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+
+  function NotGateNode(config){
+    RED.nodes.createNode(this, config);
+    this.name = config.name;
+    this.qubits = [];
+    this.targetPosition = config.targetPosition;
```

Comment:
`targetPosition` isn't being used anywhere and is redundant. But more importantly, it's also not defined as a node property in the HTML file and would never receive a value.